### PR TITLE
Added Units Test 

### PR DESCRIPTION
**Pull request by lukascorr**
* Added news tests
* coverage fixed
* composer updated
* circle ci fix
* fixs

### CODE REVIEW

File: src/Models/AfipConfig.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    public $cuit;
+
+    /* @var string */
+    public $xml_generated_directory;
+
+    /* @var string */
+    public $certificate_path;
+
+    /* @var string */
+    public $privateKey_path;
```
Comment (juanicastellan0) [inline src/Models/AfipConfig.php]: :camel: or :snake: ?

File: src/Models/AfipWebService.php [hunk 2]
```diff
@@ -48,24 +46,34 @@ class AfipWebService
      */
     protected $configuracion;
 
-    public function __construct(string $ws, array $config = [])
+    public function __construct($ws)
```
Comment (juanicastellan0) [inline src/Models/AfipWebService.php]: se puede dejar el tipado? osea: (string $ws)

File: src/Models/AfipWebService.php [hunk 3]
```diff
@@ -94,8 +102,8 @@ public function isWsOkOrFail(string $ws): bool
     public function getPosNumbers(): array
     {
         $codigos = [];
-        if ($this->service->ws == 'wsfe') {
-            $result = (new FeSinItemsParam())->FEParamGetPtosVenta($this->service->client, $this->service->authRequest);
+        if ($this->ws == 'wsfe') {
```
Comment (juanicastellan0) [inline src/Models/AfipWebService.php]: === no?

**Review by juanicastellan0 — APPROVED** [review]

File: config/config.php [hunk 4]
```diff
@@ -25,7 +25,7 @@
     ],
 
     'archivos' => [
-        'wsaaWsdl' => __DIR__ . '/../Multinexo/Afip/WSAA/wsaa.wsdl',
+        'wsaaWsdl' => getcwd() . '/src/WSAA/wsaa.wsdl',
```
Comment (pablorsk) [inline config/config.php:28]: woooooooooooooooow no conocía esa funcion!!!!!!!

Comment (lukascorr) [reply config/config.php:28]: :taco: :flushed: ? :disappointed:

**Comment by lukascorr** [comment]
Prueba

File: src/Models/AfipWebService.php (hunk 2 above)
Comment (lukascorr) [reply src/Models/AfipWebService.php]: H

Comment (lukascorr) [reply src/Models/AfipWebService.php]: dsgdsgsdg

**Pull request merged**